src/main.cpp

```diff
@@ -217,6 +219,22 @@ static enum drm_mode_generation parse_drm_mode_generation(const char *str)
 	}
 }
 
+static enum g_panel_orientation force_orientation (const char *str)
```

Comment:
Style: no space between `force_orientation` and `(` (yeah, I know we're very bad at code style, but let's try not to introduce yet another discrepancy)

Reply:
Done